### PR TITLE
Fix #457 - new united csv format with mission ID

### LINKED ISSUE #457: Update united.py to load from new CSV format
The CSV files received from United Hatzalah now has another column: the first column is the Mission ID, which will be used as the `id` field instead of the convoluted solution we used with the data and line number.
Once the script is updated, we'll remove all existing United accidents from the DB (due to the change in the `id` field) and re-load everything.

### PR DESCRIPTION
**Pull request by omerxx**
Fix #457:
* Updated import structure to DB using new united id. 
* Added fix to prevent gmail from importing irrelevant deleted files. 
* Updated sample data files to match new structure.

### CODE REVIEW

File: united.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def create_accidents(file_location):
     """
     print("\tReading accidents data from '%s'..." % file_location)
     inc = 1
```
Comment (danielhers) [inline united.py]: Is `inc` still necessary?

File: importmail.py [hunk 2]
```diff
@@ -76,6 +76,10 @@ def main(username=None, password=None, lastmail=False):
             else:
                 continue
 
+        # Handles Gmail bug which hasn't physically removed some of the deleted files
+        if mtime.month == 10 and mtime.day < 6 or mtime.month == 10 and mtime.day == 6 and mtime.hour < 10:
```
Comment (danielhers) [inline importmail.py]: Use operator `<` (http://stackoverflow.com/a/8142411/223267)

File: united.py (hunk 1 above)
Comment (omerxx) [inline united.py]: Nope. Will be removed.

**Comment by omerxx** [comment]
@danielhers - Fixed comments

**Pull request merged**